purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -173,6 +179,15 @@ open class PurchasesConfiguration(builder: Builder) {
             this.dangerousSettings = dangerousSettings
         }
 
+        /**
+         * Disable this setting if you don't want to allow pending purchases for prepaid subscriptions (only supported
+         * in Google Play). Note that entitlements are not granted until payment is done.
+         * Default is enabled.
```

Comment:
Would a developer who pays no attention to this and just updates to `8.0.0` notice any differences from this being enabled?

Reply:
Well, main difference is that users can pay for prepaid plans (if they have any) after. But in any case, entitlements won't be granted until it's payed. So it just provides more flexibility for payment for the end customer. I believe it's the most sensible default. I can't really think of why you would like to disable this, but wanted to provide the option just in case.

Reply:
Alright, yea makes sense. Thanks for the additional context! Just wanted to make sure nothing unexpected happens to developers 🙂

Reply:
Yup! It's good to double check :)